web_external/templates/widgets/leaderboard.jade

```diff
@@ -13,12 +13,17 @@ table.c-leaderboard-table
         td.c-rank= i + start
         td
           a.c-user-link(
-            user-id="#{submission.get('creatorId')}")= submission.get('creatorName')
+            href="#user/#{submission.get('creatorId')}")= submission.get('creatorName')
         td= submission.get('title') || '--'
-        td(title="#{submission.get('created')}")= girder.formatDate(submission.get('created'), girder.DATE_DAY)
+        - var date = moment.utc(submission.get('created'))
+        - var dateStr = date.format('ddd, D MMM YYYY, h:mm:ss a')
+        - var dateFromNow = date.fromNow()
+        td(title="#{dateStr} (#{date.fromNow()})")= dateStr
```

Comment:
I think you meant `dateFromNow` rather than calling `date.fromNow()` a second time.